### DISCUSSION TITLE
Welcome to VanillaWindowsReference Discussions!

### DISCUSSION THREAD

**Discussion by AndrewRathbun** [AndrewRathbun/VanillaWindowsReference#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bittib010** [comment]
why is the command using "cmd.exe /c powershell.exe"? Isnt " cmd.exe /c" unecessary in this context?

**Reply by nasbench** [reply]
It's just a habit. But you're right. The `cmd /c` can be removed in this case.
